blog/_posts/2019-11-13-affiliations-feature.md

```diff
@@ -0,0 +1,67 @@
+
+We have said this before but it certainly bears repeating.  While affiliation information is largely complete for recent refereed literature, users need to remember that not every record contains an affiliation, therefore, searching by affiliation alone will inherently be incomplete.  We strongly recommend combining affiliation searches with author searches for best results.
+
+**Using affiliations as part of your search strategy**
+
+Affiliation data for papers are available via the “Affiliations” facet in ADS.  As an example, if you search for a first author, you’ll get a list of papers having that first author, and on the left-hand side of the results page you’ll see ways to refine those results -- who their coauthors are, whether the paper is in the astronomy, physics, or general database, whether it’s a refereed publication or not, and so on.  One of those facets is “Affiliation”, and by clicking it you’ll get a list of institutional affiliations sorted by the number of times that affiliation occurs in a paper with that first author.
+
+Using one of this blog’s coauthors, searching for first_author:“Templeton, Matthew” brings up a list of his papers along with a few other “Matthew Templeton”s who are active researchers in ADS-relevant fields.  Selecting the “astronomy” collection and “limit to” yields (mostly) just his set of first-authored publications, and refereed lowers it further to a manageable number.  If you then open the Affiliations facet, you’re presented with a list of all affiliations (of any author) contained within my papers.  Matthew Templeton’s historical affiliations -- AAVSO, New Mexico State University, Yale University, Los Alamos National Laboratory -- are among the most common in the list.  However, you also find affiliations of co-authors too, for example Iowa State and the Center for Astrophysics.
+
+Affiliations in the ADS have been indexed in several different fields, with the intention of allowing multiple use cases.  We currently have assigned identifiers with parent/child relationships where a child may have multiple parents.  This has required a few modifications to remain useful. For instance, so that University of California schools can identify departments, we have assigned them a parent status, even though the "University of California System" should really be the parent level.  Likewise, NASA's Goddard Space Flight Center is at a parent level, as are France's CNRS institutions to allow for further subdivision.  Further work on a schema to allow more complex relationships between institutions is under development in conjunction with work by the ROR Community (ror.org).
```

Comment:
Explain (or provide an example of) the parent/child thing slightly earlier. Maybe "We currently have assigned identifiers with parent/child relationships, such as an academic department within a university. A child may have multiple parents, but we restrict a child from having children of its own. This has required a few modifications..."

Instead of writing out the ROR URL, hyperlink it from "ROR Community."